mysql-test/suite/tianmu/t/issue226.test

```diff
@@ -0,0 +1,5 @@
+use test;
+CREATE TABLE t1 (f1 VARCHAR(255) CHARACTER SET utf8) ENGINE=TIANMU;
+CREATE TABLE t2 AS SELECT LEFT(f1,171) AS f2 FROM t1 UNION SELECT LEFT(f1,171) AS f2 FROM t1;
```

Comment:
Good job!
It may be better to insert some data here to test  if`t2` is available or not.

Reply:
Yeah, i've done that. Let's review again.